aspnetcore/mvc/views/tag-helpers/intro.md

```diff
@@ -56,7 +56,7 @@ The `asp-for` attribute is made available by the `For` property in the [LabelTag
 
 ## Managing Tag Helper scope
 
-Tag Helpers scope is controlled by a combination of `@addTagHelper`, `@removeTagHelper`, and the "!" opt-out character.
+Tag Helpers scope is controlled by a combination of `@addTagHelper`, `@removeTagHelper`, and the "!" opt-out character. If the standard `asp-` tag helpers don't work in an [area](xref:mvc/controllers/areas), you probably need to move *Views/_ViewImports.cshtml* to the root of your project, in order for `@addTagHelper` to affect area views.
```

Comment:
Ehh, I wouldn't have this here. The section in areas is sufficient if a user is working with areas.

Reply:
I kind of agree, but this is where I looked when i had the problem. If I expand the explanation in the area docs to make it even more obvious that you need to consider moving `_ViewImports.cshtml` it will probably be sufficient.